client/src/views/pages/you-are-banned-what-now.js

```diff
@@ -0,0 +1,486 @@
+                    )}
+                  {activeBans > 0 && (
+                    <div className='text-center text-danger'><strong>You have active ban(s) therefor we highly recommend you to continue with reading!</strong></div>
+                    )}
+                    <br />
+                    <span>
+                    <h5 className='text-center'><strong>You are global banned at <br /><span className='text-primary'>Squad Community Ban List</span>, <br />what now?</strong></h5>
+                        <div className='ml-3'>When you are joining a partner organisation you may get <strong>auto-kicked</strong> with a message saying you are banned by <span className='text-primary'>Squad Community Ban List</span>. This means you passed the <strong>threshold</strong> limit of that server you joined.<br /> <br />If you want to lower your threshold you need to contact the servers you are already banned from and ask for an appeal. An alternative way would be waiting for the grace period of that server. A server owner can put a time limit, for example 365 days, and that way your bans that proceed those limits will not be count at their threshold limit and you may be able to join the server back after a while. <br /><br /><strong>Disclaimer: </strong><em>As SCBL we can't edit your ban/status at SCBL. We can only recommend you to read the rules of the servers before joining them. <br />In case you have active bans you need to contact the server you are banned and not us, since we don't act as a middleman.</em></div>
+                    </span>
+            </div>
+            <Stats step={5} {...props} steamUser={props.steamUser} />
+        </div>
+    );
+};
+
+
+const Last = (props) => {
+
+    return (
+        <div>
+            <div className={'text-center'}>
+                <h3>You are done!</h3>
+                <hr />
+                <p>We hope the previous shared information helps you resolve your issues.</p>
+                <p>If this is not the case please do contact us from our Discord server.</p>
+            </div>
+            <Stats step={6} {...props} />
+        </div>
+    );
+};
+
+
+export default function (props) {
+    const search = props.match.params.uid;
+    let isRedirected = false;
+    if(search === '11111111111111111') {
+        isRedirected = true;
+    }
+    const isValidSteam64ID = search && search.match(/^[0-9]{17}$/);
+
+    const {data} = isValidSteam64ID
+    ? useQuery(GET_PLAYER, { variables: { id: search } })
+    : { data: null };
+
+  return (
+    <Layout>
+      <section className="section section-lg pt-lg-0 mt--200">
+        <Container>
+          <Card className="shadow border-0">
+            <CardBody className="pt-5 pb-2 border-bottom">
+              <div className="icon icon-shape bg-gradient-warning rounded-circle text-white mb-4">
+                <i className="fa fa-life-ring" />
+              </div>
+              <h6 className="text-warning text-uppercase">SCBL For Dummies</h6>
```

Comment:
Like I mentioned in Discord I'm not sure this is the best name. Those that don't know about the "x for Dummies" trend based on the books might take offence to this, which is the last thing we want to do when players reading this are probably going to easily get annoyed if they're banned.

"I'm banned, what now?" is probably better - like you have in the nav.

Reply:
Done